File: solucoes/2015/nivel jr/fase 2/js/codigo.js
N, C
var code = []
scanf("%d", "N")
var qtd = 0

for(var i = 0; i < N; i++){
    scanf("%d", "C")
    code.push(C)
}

for(var i = 0; i < N; i++){
    if(code[i] == 1 && code[i + 1] == 0 && code[i + 2] == 0){
        qtd += 1
    }
    if(i == N - 3){
        break
    }
}

printf("%d", qtd)

File: solucoes/2015/nivel jr/fase 2/py/codigo.py
N = int(input())
C = list(map(int, input().split()))

qtd = 0

for i in range(N):
    if C[i] == 1 and C[i + 1] == 0 and C[i + 2] == 0:
        qtd += 1
    if i == N - 3:
        break
print(qtd)